Workbooks/WorkbooksMetadata.json

```diff
@@ -892,5 +892,18 @@
     "templateRelativePath": "WebApplicationFirewallWAFTypeEvents.json",
     "subtitle": "",
     "provider": "Microsoft"
+  },
+  {
+    "workbookKey": "Citrix_ADC_WAF",
+    "logoFileName": "citrix_logo.svg",
+    "description": "Gain insight into the Citrix ADC WAF logs",
+    "dataTypesDependencies": [ "CommonSecurityLog" ],
+    "dataConnectorsDependencies": [ "Citrix" ],
```

Comment:
Please reflect the data connector updated id value here.

Reply:
Done